### PR TITLE
Update README.md with a few install changes

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweney@redhat.com>

Testing found a few nits to clean up in the install process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +52,7 @@ Then to install buildah follow the steps in this example:
   git clone https://github.com/projectatomic/buildah ./src/github.com/projectatomic/buildah 
   cd ./src/github.com/projectatomic/buildah 
   make 
+  make install
   ./buildah --help
```
Comment (nalind) [inline README.md]: After _make install_, do we still need to run the tool from the current directory?  Not a blocker either way.

**Comment by nalind** [comment]
LGTM.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ 88dc284

File: README.md (hunk 1 above)
Comment (TomSweeneyRedHat) [inline README.md]: no we don't, I'll touch that up.

**Pull request closed without merging**